core/src/cluster_info_vote_listener.rs

```diff
@@ -434,19 +443,22 @@ impl ClusterInfoVoteListener {
         subscriptions: &RpcSubscriptions,
         epoch_stakes: Option<&EpochStakes>,
         verified_vote_sender: &VerifiedVoteSender,
+        replay_votes_receiver: &ReplayVotesReceiver,
     ) -> Result<()> {
-        let timer = Duration::from_millis(200);
+        let timer = Duration::from_millis(100);
         let mut vote_txs = vote_txs_receiver.recv_timeout(timer)?;
```

Comment:
How about using a [`select_timeout`](https://docs.rs/crossbeam-channel/latest/crossbeam_channel/struct.Select.html#method.select_timeout) to fire when a vote is received on either receiver? Then you don't have to worry about waiting too long (`200ms` vs `100ms`) in between channel notifications

Reply:
awesome, didn't even know that existed, thanks!

Reply:
Switched to using `Select::ready_timeout`